server/utils/hooks.py

```
@@ -41,11 +41,23 @@ def _default_before_add_member(org_id: str, current_member_count: int) -> Tuple[
     return True, None
 
 
+def _default_report_usage(org_id: Optional[str], usage_usd: float, metadata: dict) -> None:
+    """Called after LLM usage is recorded. For marketplace metering integration."""
+    pass
+
+
+def _default_verify_entitlement(org_id: Optional[str]) -> Tuple[bool, Optional[str]]:
```

Ruff (0.15.17)

[warning] Unused function argument: `org_id`

(ARG001)